README.md

```diff
@@ -1,46 +1,62 @@
 
-### `npm run build`
+#### Visor (non-tabbable)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+It displays the current expression, fed from the store, and the calculated sum. Calculations are made in real time. It should clearly display (through colors and visually) when the expression is malformed.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+The visor is not interactive and non tabbable. Clicking over it **does not allow** the user to set a cursor position for editing the expression. That would create a complexity that is not in the main focus of this project.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+When tabbed to the number group, 5 is selected by default. When tabbing to the secondary group, save is selected by default.
 
-### `npm run eject`
+#### Scope (tabbable)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+This is a scrollable list of precomputated values and operations. It holds 26 values, each associated with a letter, so typing A-Z will recall the value inside the current expression.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+The items in scope are fed from the store (configurable in the History page). Each expression saved in the current session automatically shows in scope. Each item displays its shortcut letter, the expression (or its label) and the computed result.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+The Scope is the stretchable part of the calculator, adapting to the window's height.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+#### Navigation
 
-## Learn More
+Contains three links, one for the calculator (main) page, one for dev, and one for history. It also indicates the current active page.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Dev (page)
```

Comment:
Shouldn't this be called "About"?

Reply:
It could and probably should. The case for dev is:
"Dev" implied "if you're not interested in development - this is not for you". "About" implies there's something for anyone to appreciate there - but it mostly describes the tech behind it, right?
Having said that, "about" sounds more common, and common is better. Yep, about sounds better.